packages/generator-nitro/generators/app/templates/src/ui.ts

```diff
@@ -2,6 +2,10 @@
 
 <% if (options.exampleCode) { %>import { hot } from '@gondel/plugin-hot';
 
+import 'core-js/features/array/includes';
```

Comment:
I think it would make more sense to import the polyfills where they are needed.
(e.g. array/includes and string/starts-with in example pattern)